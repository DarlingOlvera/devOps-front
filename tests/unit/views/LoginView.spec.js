import { shallowMount } from "@vue/test-utils";
import Login from "@/views/LoginView.vue"

//importante mockear vuetify
jest.mock('vuetify', () => ({
  useDisplay: jest.fn(() => ({ xs: true })),
}));

jest.mock('vue-router',()=>({
  useRouter: jest.fn(()=>({router:'home'}))
}))


describe('Pruebas en LoginView', () => {
  test('El html debe coincidir con el snaptshot', () => {
    const wrapper = shallowMount(Login)

    //expect(wrapper.html()).toMatchSnapshot()
  });
});
