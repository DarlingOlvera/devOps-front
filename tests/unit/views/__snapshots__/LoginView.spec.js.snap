// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en LoginView El html debe coincidir con el snaptshot 1`] = `
<v-container fluid="">
  <v-row class="d-flex justify-center my-10">
    <h3 class="text-h2 text-title">Iniciar Sesi√≥n</h3>
  </v-row>
  <v-row class="d-flex justify-center mt-15">
    <v-card class="pa-10 rounded-xl elevation-0" width="100%" max-width="500" height="auto">
      <form-stub as="form" validationschema="[object Object]" validateonmount="false" keepvalues="false" name="loginForm"></form-stub>
    </v-card>
  </v-row>
</v-container>
`;
